src/ML-POD/pair_pod.h

```diff
@@ -0,0 +1,125 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/ Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef PAIR_CLASS
+// clang-format off
+PairStyle(mlpod,PairPOD);
```

Comment:
When you change the name of the pair style, all "derived" names (name of source files, name of Class, include guards, name of documentation file, etc.) should be changed as well so that they are consistent.

Reply:
> > @exapde @rohskopf I have a little extra time this week. So I could help implementing some of the suggested changes.
> 
> I have submitted a pull request in the fork with a set of (simple) changes.

Hi @ akohlmey! Thanks for your help. It is much  appreciated.

Reply:
Changed the pair style name from pod to mlpod and updated POD's documentation files to reflect the name change. Changed the example folder name from pod to mlpod.